docs/resources/dms_rabbitmq_instance.md

```diff
@@ -0,0 +1,175 @@
+---
+subcategory: "Distributed Message Service (DMS)"
+---
+
+# huaweicloud\_dms\_rabbitmq\_instance
+
+## Example Usage
+
+### Basic Instance
+
+```hcl
+data "huaweicloud_dms_az" "test" {}
+
+data "huaweicloud_vpc" "test" {
+  name = "vpc-default"
+}
+
+data "huaweicloud_vpc_subnet" "test" {
+  name = "subnet-default"
+}
+
+data "huaweicloud_dms_product" "test" {
+  engine        = "rabbitmq"
+  instance_type = "cluster"
+  version       = "3.7.17"
+}
+
+resource "huaweicloud_networking_secgroup" "test" {
+  name        = "secgroup_1"
+  description = "secgroup for rabbitmq"
+}
+
+resource "huaweicloud_dms_rabbitmq_instance" "test" {
+  name              = "instance_1"
+  description       = "rabbitmq test"
+  access_user       = "user"
+  password          = "Rabbitmqtest@123"
+  vpc_id            = data.huaweicloud_vpc.test.id
+  network_id        = data.huaweicloud_vpc_subnet.test.id
+  security_group_id = huaweicloud_networking_secgroup.test.id
+  available_zones   = [data.huaweicloud_dms_az.test.id]
+  product_id        = data.huaweicloud_dms_product.test.id
+  engine_version    = data.huaweicloud_dms_product.test.version
+  storage_space     = data.huaweicloud_dms_product.test.storage
+  storage_spec_code = data.huaweicloud_dms_product.test.storage_spec_code
+
+  tags = {
+    key   = "value"
+    owner = "terraform"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) The region in which to create the DMS rabbitmq instance resource.
+  If omitted, the provider-level region will be used. Changing this creates a new instance resource.
+
+* `name` - (Required, String) Specifies the name of the DMS rabbitmq instance. An instance name starts with a letter,
+    consists of 4 to 64 characters, and supports only letters, digits, hyphens (-) and underscores (_).
+
+* `description` - (Optional, String) Specifies the description of the DMS rabbitmq instance. It is a character
+    string containing not more than 1024 characters.
+
+* `engine_version` - (Required, String, ForceNew) Specifies the version of the rabbitmq engine.
+    Valid value is "3.7.17".
+    Changing this creates a new instance resource.
+
+* `storage_space` - (Required, Int, ForceNew) Specifies the message storage space. Value range:
+    - Single-node RabbitMQ instance: 100–90000 GB
+    - Cluster RabbitMQ instance: 100 GB x Number of nodes to 90000 GB, 200 GB x Number of nodes to 90000 GB, and 300 GB x Number of nodes to 90000 GB
+    
+    Changing this creates a new instance resource.
+
+* `storage_spec_code` - (Required, String, ForceNew) Specifies the storage I/O specification. Value range:
+    - dms.physical.storage.normal
```

Comment:
seems "dms.physical.storage.normal" is not supportted in the console, maybe sold out.

Reply:
fixed